api/src/main/generated/org/pmiops/workbench/utils/WorkspaceMapperImpl.java

```diff
@@ -0,0 +1,129 @@
+package org.pmiops.workbench.utils;
+
+import java.util.HashSet;
+import java.util.List;
+import javax.annotation.Generated;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.firecloud.model.FirecloudWorkspace;
+import org.pmiops.workbench.model.ResearchPurpose;
+import org.pmiops.workbench.model.SpecificPopulationEnum;
+import org.pmiops.workbench.model.UserRole;
+import org.pmiops.workbench.model.Workspace;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+import org.springframework.stereotype.Component;
+
+@Generated(
```

Comment:
The mapper impl files are generated at compile time, and should not be checked into source control.

Reply:
something is clearly broken with gitignore.